package.json

```diff
@@ -7,6 +7,8 @@
     "prestart": "npm ci",
     "start": "pm2 flush && pm2 start app.js --name OctoFarm -e ./logs/pm2.error.log -o ./logs/pm2.out.log --time",
     "stop": "pm2 stop OctoFarm",
+    "restart": "pm2 stop OctoFarm && pm2 start OctoFarm",
+    "update": "git update-index --assume-unchanged ./config/db.js && git pull && npm ci && npm run restart",
```

Comment:
Since I updated instructions on OF readme to use `.env` it seems more logical to use .env for database now. 
We can write a startup script to create the .env file when it doesnt exist which copies the MongoUri from the config/db.js file, so people dont have to worry about that. In case .env already exists and has a MONGO variable, we can leave it safely as is.
I think this npm script kinda stuff is a bad idea, especially since it does not fix the problem that the file still causes git changes unless the update command is called once... please no. Instead we need to have something more robust and self-healing rather than put into custom git commands, which are quite easy to get wrong.
Use gitignore properly: ignore, untrack, and self-heal with a smart startup js/ts script which is called by our application.

Reply:
That wasn't the intention. The purpose of this was to lay the ground work that will support users now and allow us breathing room to update and change how we see fit in the future.